test/server_search.cpp

```diff
@@ -692,11 +692,11 @@ struct TestData
     std::string url = "/ROOT/search?" + query;
 
     if ( start >= 0 ) {
-      url += "&start=" + to_string(start);
+      url += "&start=" + std::to_string(start);
```

Comment:
I think all `to_string()`s in this file are supposed to be `kiwix::to_string()` from `src/tools/stringTools.h`. https://github.com/kiwix/libkiwix/blob/b9e03d27726617c8c3305eede44f71df7f28e031/src/tools/stringTools.h#L56-L62

Though if the need for `kiwix::to_string()` has disappeared (since the platforms that didn't provide `std::to_string()` are no longer supported) we better drop it.

Reply:
It is supposed to be `std::to_string()` has it is this namespace that was used by default here.
But indeed, we should simply remove `kiwix::to_string()` (will do it in this PR)